src/components/Header.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useContext } from 'react';
+import { Form } from './Form';
+
+import cn from 'classnames';
+
+import { TodoContext } from '../store/TodoContext';
+
+const Header: React.FC = () => {
+  const { todos, changeCompleteAll, completedTodosLength } =
+    useContext(TodoContext);
+
+  return (
+    <header className="todoapp__header">
+      {!!todos.length && (
+        <button
+          onClick={() => changeCompleteAll()}
```

Comment:
```suggestion
          onClick={changeCompleteAll}
```

can be a bit simplify